lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1190,6 +1190,62 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
     return;
   }
 
+  // Define the function
+  auto addPushConstArgs = [&](const ResourceNode &node) {
```

Comment:
This lambda is very long. How about making a freestanding or a member function?

Reply:
I make it a member function. Thanks.